tofu/tests/tests09_tutorials/tuto_plot_basic.py

```diff
@@ -9,10 +9,6 @@
 # The following imports matplotlib, preferably using a
 # backend that allows the plots to be interactive (Qt5Agg).
 import matplotlib
-try:
-    matplotlib.use('Qt5Agg')
```

Comment:
Removed because we don't want tofu to mess up with the backend at all, right ?

Reply:
because this is a unit test. The real tutorial is in the `examples` directory. Here we just want to check it runs, so no need to plot.

Reply:
before this was being ignored because we hard coded the backend to Agg for tests.